docarray/array/mixins/attribute.py

```diff
@@ -28,3 +45,24 @@ def _get_documents_attribute(
             )
         else:
             return [getattr(doc, field) for doc in self]
+
+    def _set_array_attribute(
+        self,
+        field: str,
+        values: Union[List, AbstractDocumentArray, 'TorchTensor', 'NdArray'],
+    ):
+        """Set all document if this DocumentArray with the passed values
```

Comment:
```suggestion
        """Set all Documents in this DocumentArray using the passed values
```
Please confirm correct. Odd wording before.